draft-ietf-dnsop-svcb-https.md

```diff
@@ -166,22 +160,21 @@ a non-normative manner.  (As mentioned above, this all
 applies equally to the HTTPS RR which shares
 the same encoding, format, and high-level semantics.)
 
-The SVCB RR has two forms: AliasForm, which aliases a name to another name,
-and ServiceForm, which provides connection information bound to a service
+The SVCB RR has two modes: AliasMode, which aliases a name to another name,
+and ServiceMode, which provides connection information bound to a service
 endpoint domain.  Placing both forms in a single RR type allows clients to
 fetch the relevant information with a single query.
 
 The SVCB RR has two mandatory fields and one optional.  The fields are:
 
-1. SvcFieldPriority: The priority of this record (relative to others,
-   with lower values preferred).  A value of 0 indicates AliasForm.
+1. SvcPriority: The priority of this record (relative to others,
+   with lower values preferred).  A value of 0 indicates AliasMode.
    (Described in {{pri}}.)
-2. SvcDomainName: The domain name of either the alias target (for
-   AliasForm) or the alternative service endpoint (for ServiceForm).
-3. SvcFieldValue (optional): A list of key=value pairs
-   describing the alternative service endpoint for the domain name specified in
-   SvcDomainName (only used in ServiceForm and otherwise ignored).
-   Described in {{svcfieldvalue}}.
+2. TargetName: The domain name of either the alias target (for
+   AliasMode) or the alternative service endpoint (for ServiceMode).
+3. SvcParams (optional): A list of key=value pairs
+   describing the alternative service endpoint at
+   TargetName (only used in ServiceMode and otherwise ignored).
```

Comment:
```suggestion
3. SvcParams (optional): A list of key=value pairs
   describing the alternative endpoint at
   TargetName (only used in ServiceMode and otherwise ignored). 
   Described in {{svcparams}}.
```